net/iputils/Makefile

```diff
@@ -0,0 +1,151 @@
+#
+# Copyright (C) 2019 Noah Meyerhans <frodo@morgul.net>
+#
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=iputils
+PKG_VERSION:=20190709
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Noah Meyerhans <frodo@morgul.net>
+PKG_LICENSE:=BSD-3
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE:=s$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/iputils/iputils/archive
+PKG_HASH:=a15720dd741d7538dd2645f9f516d193636ae4300ff7dbc8bfca757bf166490a
+
+PKG_INSTALL:=
+PKG_USE_MIPS16:=0
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/iputils-s$(PKG_VERSION)
+MESON_BUILD_DIR:=$(PKG_BUILD_DIR)/openwrt-build
+
+PKG_BUILD_DEPENDS:=meson/host
+include $(INCLUDE_DIR)/package.mk
+include ../../devel/meson/meson.mk
+MESON_ARGS:=-Dprefix=/usr -DBUILD_MANS=false -DBUILD_HTML_MANS=false -DUSE_CAP=false
+
+define Build/Configure
+  $(call Build/Configure/Meson)
+endef
+
+define Package/iputils/Default
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=iputils
+  URL:=https://github.com/iputils/iputils
+endef
+
+
+### ping
```

Comment:
Can be this removed?

Reply:
It's a comment, so it certainly _can_. I find it helpful to delineate the sections of the makefile related to the different packages being defined. My preference would be to expand on the comment, rather than delete it.